assets/js/hooks/useUpdateAnimation.tsx

```diff
@@ -21,7 +21,7 @@ export const useUpdateAnimation = (
       timer.current = undefined;
       setShowAnimation(false);
     }, 2000);
-  }, deps);
+  }, deps); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Can we do something like `[prevValue, showAnimationOnMount].concat(deps)` here? Or else add a comment explaining why we have to ignore the warning / why it's safe to do this.

Reply:
So there will be a warning about the dependencies array not being statically analyzable either way. I feel like there's probably a better way to achieve this though. I'll add a comment for now.

Reply:
Hm, ok. I did just notice a [remark](https://github.com/facebook/react/tree/main/packages/eslint-plugin-react-hooks#advanced-configuration) in this package's docs that _"we recommend most custom Hooks to not use the dependencies argument, and instead provide a higher-level API that is more focused around a specific use case."_ So there probably is some better way we could be doing this, but that doesn't have to be investigated here.